moveit_core/planning_scene/src/planning_scene.cpp

```diff
@@ -1807,25 +1790,31 @@ bool PlanningScene::processCollisionObjectMove(const moveit_msgs::CollisionObjec
       ROS_WARN_NAMED(LOGNAME, "Move operation for object '%s' ignores the geometry specified in the message.",
                      object.id.c_str());
 
-    const Eigen::Isometry3d& t = getTransforms().getTransform(object.header.frame_id);
+    const Eigen::Isometry3d& world_to_object_header_transform = getTransforms().getTransform(object.header.frame_id);
+    Eigen::Isometry3d object_header_to_pose_transform;
+                                                      
+    PlanningScene::poseMsgToEigen(object.pose, object_header_to_pose_transform);
+
+    const Eigen::Isometry3d& object_frame_transform =
```

Comment:
Same comment as above on this one, suggest dropping the reference here.

Reply:
Done